oaknational/Oak-Web-Application
# Chore/lesq 1067/update meta descriptions [LESQ-2067]

## Description

Music year: 1985

- Update meta seo descriptions on the teachers tab and keystage subject pages
- Also updated default homepage SEO in config

## How to test

1. Go to https://deploy-preview-2802--oak-web-application.netlify.thenational.academy
2. meta description should be "Browse and download our lesson planning resources, view our curriculum plans and explore our AI lesson assistant - all optional, free and adaptable."
3. Go to /teachers
4. meta title should be 'Free, time-saving teacher resources | Oak National Academy', meta description should be 'Explore our free, time-saving teacher resources from Oak National Academy. Browse and download worksheets, quizzes and slides from KS1 to KS4. '
5. Go to /teachers/key-stages/ks2/subjects
6. meta description should be 'Click here to browse and download our free KS2 teaching resources for lesson planning. Our teaching resources are made by subject experts and entirely free to download and use.'

## Checklist
Meta descriptions should be correct on:
- [x] Homepage (AI)
- [x] Teachers tab
- [x] subject listing pages
